exec.go

```diff
@@ -162,7 +162,10 @@ func execProcess(context *cli.Context) (int, error) {
 	if err != nil {
 		return -1, err
 	}
-	bundle := utils.SearchLabels(state.Config.Labels, "bundle")
+	bundle, ok := utils.SearchLabels(state.Config.Labels, "bundle")
+	if !ok {
+		return -1, errors.New("bundle not found in labels")
+	}
 	p, err := getProcess(context, bundle)
```

Comment:
Are empty values allowed here for `bundle`? (Wondering if `""` is ok to pass, so if there's an actual difference between "not set" and "no value" here)

Reply:
It's a path to container root directory, and if empty, getProcess will return an error (from chdir). I do not think we need some additional checks here.

OTOH it make sense to move this code into getProcess as it is only needed if `--process` is not given.

Reply:
... but I am going to address that separately.

PTAL @thaJeztah